src/libs/actions/Report.ts

```diff
@@ -1538,6 +1538,7 @@ function editReportComment(reportID: string, originalReportAction: OnyxEntry<Rep
                     text: reportComment,
                 },
             ],
+            lastModified: DateUtils.getDBTime(),
```

Comment:
Don't we need to revert this in the failure data?

Reply:
I realized that the failureData already has the previous report action, so I think no need to add `lastModified: originalReportAction.lastModified,` again.
https://github.com/Expensify/App/blob/9d8e7feefd8e588d4a7bc8f53035fe0b924ad618/src/libs/actions/Report.ts#L1566-L1574

Reply:
Ohhhh ok yeah, that makes sense